eth-pm/src/App.tsx

```diff
@@ -67,13 +68,13 @@ const useStyles = makeStyles({
 });
 
 function App() {
-  const [waku, setWaku] = useState<WakuPrivacy>();
+  const [waku, setWaku] = useState<RelayNode>();
   const [provider, setProvider] = useState<Web3Provider>();
   const [encryptionKeyPair, setEncryptionKeyPair] = useState<
     KeyPair | undefined
   >();
   const [privateMessageDecoder, setPrivateMessageDecoder] =
-    useState<AsymDecoder>();
+    useState<IDecoder<DecodedMessage>>();
```

Comment:
This is not good:
- Complex type
- Does not provide access to the full API of the `Decoder` (e.g. access to signature).

Need to expose the type in js-waku to fix this (wip).